ezcontent_theme.theme

```diff
@@ -67,3 +67,20 @@ function ezcontent_theme_preprocess_menu_local_action(&$variables) {
     '#url' => $link['url'],
   ];
 }
+
+function ezcontent_theme_preprocess_block(&$variables) {
+  $variables['site_name'] = \Drupal::config('system.site')->get('name');
+}
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function ezcontent_theme_theme_suggestions_block_alter(&$suggestions, $variables) {
+  $content = $variables['elements']['content'];
+  $content_layout = $variables['elements']['content']['content'];
```

Comment:
In some cases where the nested content variable is not preset, this line of code would generate notices in database logs. Please add a condition here to check if the nested content variable is present.

`isset($variables['elements']['content']['content']) ? $variables['elements']['content']['content'] : [];`